rrrspec-client/lib/rrrspec/client/rspec_runner_factory.rb

```diff
@@ -0,0 +1,24 @@
+require 'rrrspec/client/rspec_runner'
+
+module RRRSpec
+  module Client
+    class RSpecRunnerFactory
+      attr_reader :next_factory
```

Comment:
A registry would be a nice solution, but it would limit how the choice of the runner is made. 

Currently the choice is made in the plugin, if we change it to a registry, the choice would be made be rrrspec and not in the plugin. 

My doubt is if we can find a general rule that works for all future cases so that the choice can be made in rrrspec code and not in the plugin.